src/eos-google-chrome

```diff
@@ -94,6 +140,12 @@ class GoogleChromeLauncher:
         except OSError as e:
             exit_with_error("Could not launch Chrome: {}".format(repr(e)))
 
+        self._wait_for_installation()
+        logging.info("Chrome successfully installed")
+
+        self._post_install_chrome()
```

Comment:
Same here: I don't think this logic belongs in this script, which should exit ASAP after handing control over to GS.